Include/Rover/ListTrial.hpp

```diff
@@ -16,7 +17,7 @@ namespace Rover {
       using Sample = S;
 
       //! The type of the constant iterator.
-      using ConstIterator = typename std::vector<Sample>::const_iterator;
+      using Iterator = typename TrialIterator<ListTrial>;
```

Comment:
Remove the ```typename```.

Reply:
It may be better to stick to using ```vector::iterator``` for this class.

Reply:
I don't know, no real difference in compiled code. Using `TrialIterator` feels a bit more "canonical".

Reply:
Okay, I was concerned it would introduce additional overhead, but if it ends up being the same then feel free to stick to using ```TrialIterator```.